Gemfile.lock

```diff
@@ -405,69 +409,68 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
```

Comment:
### [large_object_store](https://github.com/anamartinez/large_object_store)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.3.4 → 1.7.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/large_object_store/1.3.4/1.7.0)_]





<details>
<summary>Commits</summary>

A change of **31** commits. See the full changes on [the compare page](https://github.com/zendesk/large_object_store/compare/v1.3.4...v1.7.0).

These are the first 10 commits:
- (eb5112c) [add CI](https://github.com/zendesk/large_object_store/commit/eb5112cb87b5e3077d3eee1c538b13d49974612c)
- (216095e) [Merge pull request #16 from anamartinez/grosser/ci](https://github.com/zendesk/large_object_store/commit/216095e1c3d43327921e8c025bcdfb0afde84c74)
- (7438aa1) [Add support for custom serializer](https://github.com/zendesk/large_object_store/commit/7438aa1af250a7e76c72fdd8d014efc9c3f55523)
- (962fe4c) [Merge pull request #14 from ggrossman/ggrossman/custom_seria…](https://github.com/zendesk/large_object_store/commit/962fe4c85c12aa41dc11dc11d726b338aacb516c)
- (45e0b84) [v1.4.0](https://github.com/zendesk/large_object_store/commit/45e0b84e1e6b3d826364057922d148704414e0c6)
- (aa537a4) [CI: Update config](https://github.com/zendesk/large_object_store/commit/aa537a4e7361bd5626c1a2b1994d0d66713229e3)
- (9b389e0) [Simplify CI branch protection](https://github.com/zendesk/large_object_store/commit/9b389e00a27044beb3f46566bb89aa719060deb3)
- (7bdc7fe) [Merge pull request #18 from zendesk/bquorning.simplify-ci-br…](https://github.com/zendesk/large_object_store/commit/7bdc7fecd4aebfcb420fd0efacfd272c6174a651)
- (35d5363) [Convert specs to RSpec 2.99.2 syntax with Transpec](https://github.com/zendesk/large_object_store/commit/35d53636cecb880001dfcdbe355a520fcbe1dc51)
- (5867502) [Upgrade bundled gems](https://github.com/zendesk/large_object_store/commit/5867502b2029bd38a55056bea0c832fbe8c5311d)


</details>